File: src/jaff/templates/fortran_dlsodes/fluxes.f90
module fluxes
    use commons
    use reactions
    contains

    function get_fluxes(y, tgas, crate, av) result(flux)
        implicit none
        real*8::flux(nreactions)
        real*8,intent(in)::y(nspecs), tgas, crate, av
        real*8::k(nreactions)

        k = get_reactions(tgas, crate, av)

        !! PREPROCESS_FLUXES

        !! PREPROCESS_END

    end function get_fluxes

end module fluxes

File: src/jaff/templates/python_solve_ivp/fluxes.py
import numpy as np
from commons import *
from rates import get_rates


def get_fluxes(y, tgas, crate, av):
    k = get_rates(tgas, crate, av)

    flux = np.zeros_like(y)

    # PREPROCESS_FLUXES

    # PREPROCESS_END

    return flux
